src/components/404.vue

```diff
@@ -24,4 +24,8 @@ export default {
   font-size: 2em;
   font-weight: 100;
 }
+
+.box {
+  text-align: center;
+}
```

Comment:
May be you can add this to a global css file

Reply:
Maybe you didn't notice that there is a ' . ' before 'box'
this don't need a fix